lib/rex/ui/text/irb_shell.rb

```diff
@@ -28,6 +28,7 @@ def run
 
       IRB.setup(nil)
       IRB.conf[:PROMPT_MODE]  = :SIMPLE
+      IRB.conf[:USE_MULTILINE] = true
```

Comment:
I am using this to make sure that IRB uses Reline, allowing us to set `input_library: :reline` in calls to `with_context` when interacting with the history manager.